templates/default/appserver.nginx.conf.erb

```diff
@@ -4,6 +4,15 @@
 upstream <%= @name %>_<%= @application[:domains].first %> {
   server unix:<%= @deploy_dir.to_s %>/shared/sockets/<%= @name %>.sock fail_timeout=0;
 }
+
+server {
+  listen 80;
+  listen [::]:80;
+  server_name _;
+
+  return 301 https://$host$request_uri;
+}
```

Comment:
The ssl termination happens on load balancer, which is put in front of nginx. It means that this block will match all the requests coming from load balancer and redirect them, and redirected request will again end up processed by this block. I just tried this configuration on eado and it indeed creates an infinite redirect loop. Let's try to use load balancer based redirect tomorrow.

Reply:
Oh, right. :man_facepalming: